### ISSUE TITLE
Port is lost from host header

### ISSUE THREAD

**Issue by trygve-lie** [jkyberneees/fast-proxy#6]
I am not 100% sure what correct behavior should be, but the `host` on the header of a proxied request looses its `port`.

Proxy server:

```js
const { proxy } = require('fast-proxy')({
    base: 'http://localhost:9600'
})
const gateway = require('restana')()

gateway.all('/', (req, res) => {
    proxy(req, res, req.url, {})
})

gateway.start(8080)
```

Target server:

```js
const server = require('restana')();
server.get('/', (req, res) => {
    console.log(req.headers.host);
    res.send('Hello from simple http service');
});
server.start(9600);
```

When the proxy server is requested and proxied to the target server, the target server will print just `localhost`. I expect it to print `localhost:8080`.

Is current behavior correct or should the `port` be forwarded too?

It would also be nice to have a way to override the upstream http headers from the proxy to the target.

I'm happy to provide a PR to fix the missing port or add a solution to be able to set http headers on the upstream request.

**Comment by jkyberneees** [comment]
Hi @trygve-lie thanks for reporting this bug. Will be fixed in v1.4.0

**Comment by jkyberneees** [comment]
v1.4.0 of fast-proxy was released.
v2.1.0 of fast-gateway was released. 

Thanks again for reporting. Closing this issue.
